### PR TITLE
Update to Cadence v0.26.0 and Go SDK v0.27.0

### PR DESCRIPTION
**Pull request by turbolent**
- Update to [Cadence v0.26.0](https://github.com/onflow/cadence/releases/tag/v0.26.0)
- Update to [Go SDK v0.27.0](https://github.com/onflow/flow-go-sdk/releases/tag/v0.27.0)

### CODE REVIEW

**Review by SaveTheRbtz — APPROVED** [review]

**Review by fxamacker — APPROVED** [review]

File: fvm/transactionInvoker.go [hunk 1]
```diff
@@ -80,30 +81,34 @@ func (i *TransactionInvoker) Process(
 
 	env := NewTransactionEnvironment(*ctx, vm, sth, programs, proc.Transaction, proc.TxIndex, span)
 
-	predeclaredValues := valueDeclarations(env)
-
 	location := common.TransactionLocation(proc.ID)
 
+	runtimeEnv := runtime.NewBaseInterpreterEnvironment(runtime.Config{})
+	declareStandardLibraryValues(runtimeEnv, env)
```
Comment (turbolent) [inline fvm/transactionInvoker.go:87]: A great optimization opportunity would be to reuse this environment across multiple transactions (and also for the contract invocations). Likewise, for script executions we should create and reuse a single script environment.

Comment (SaveTheRbtz) [reply fvm/transactionInvoker.go:87]: cc: @pattyshack

Comment (pattyshack) [reply fvm/transactionInvoker.go:87]: creating a new env is relatively cheap after moving out the meter parameters stuff.  are we reusing the standard lib values on cadence side (cuz re-creating those are expensive if i recall correctly)?

**Review by pattyshack — APPROVED** [review]

File: cmd/util/ledger/reporters/fungible_token_tracker_test.go [hunk 2]
```diff
@@ -3,6 +3,7 @@ package reporters_test
 import (
 	"encoding/hex"
 	"fmt"
+	"github.com/onflow/cadence/runtime"
```
Comment (janezpodhostnik) [inline cmd/util/ledger/reporters/fungible_token_tracker_test.go:6]: There are some linting issues.

**Review by janezpodhostnik — CHANGES_REQUESTED** [review]
Looks good, but there are some lint issues. And also integration tests seem to have problems with this change.

**Comment by turbolent** [comment]
@janezpodhostnik integration code is building with many unrelated errors, even after merging `master`. No clue what is going on

File: fvm/transactionInvoker.go [hunk 3]
```diff
@@ -80,30 +82,34 @@ func (i *TransactionInvoker) Process(
 
 	env := NewTransactionEnvironment(*ctx, vm, sth, programs, proc.Transaction, proc.TxIndex, span)
 
-	predeclaredValues := valueDeclarations(env)
-
 	location := common.TransactionLocation(proc.ID)
 
+	runtimeEnv := runtime.NewBaseInterpreterEnvironment(runtime.Config{})
```
Comment (pattyshack) [inline fvm/transactionInvoker.go:87]: can you add `// TODO(patrick): reuse runtimeEnv`

**Comment by turbolent** [comment]
I keep on re-running the unit tests, but they fail each time due to flaky tests :(

**Comment by pattyshack** [comment]
bors merge

**Review by SaveTheRbtz — APPROVED** [review]

**Comment by SaveTheRbtz** [comment]
bors merge

**Comment by SaveTheRbtz** [comment]
bors merge

**Comment by SaveTheRbtz** [comment]
bors merge

**Pull request merged**